conjur/controller/logout_controller.py

```diff
@@ -31,17 +32,18 @@ def remove_credentials(self):
         """
         Method for removing credentials from user machine
         """
-        logging.debug("Attempting to log out of Conjur")
         try:
-            if not os.path.exists(DEFAULT_NETRC_FILE):
-                sys.stdout.write("Successfully logged out from Conjur.\n")
-            elif os.path.exists(DEFAULT_NETRC_FILE) and os.path.getsize(DEFAULT_NETRC_FILE) != 0:
-                conjurrc = ConjurrcData.load_from_file(DEFAULT_CONFIG_FILE)
-                self.logout_logic.remove_credentials(conjurrc.conjur_url)
-                logging.debug("Logout successful")
-                sys.stdout.write("Successfully logged out from Conjur.\n")
-            else:
-                raise Exception("You are already logged out")
+            loaded_conjurrc = ConjurrcData.load_from_file(DEFAULT_CONFIG_FILE)
+            self.logout_logic.remove_credentials(loaded_conjurrc)
+        # Catches when credentials do not exist in the keyring. If the key does not exist,
+        # the user has already logged out
+        except keyring.errors.PasswordDeleteError:
+            # pylint: disable=logging-fstring-interpolation
+            logging.debug("Successfully removed credentials from the "
+                          f"keyring '{keyring.get_keyring().name}'")
```

Comment:
Should be catched inside remove_credentials I think or even deeper.

Reply:
moved this to the adapter